packages/components/addon/components/hds/pagination-bar/total-count.js

```diff
@@ -0,0 +1,40 @@
+import Component from '@glimmer/component';
+
+export default class HdsPaginationBarIndexComponent extends Component {
+  // totalPages = Math.max(
+  //   Math.ceil(this.args.totalItems / this.itemsPerPage),
+  //   1
+  // );
+
+  /**
+   * Shows totalItems if true
+   *
+   * @param hasTotalItems
+   * @type {boolean}
+   * @default true
+   */
+  get hasTotalItems() {
+    return this.args.hasTotalItems ?? true;
```

Comment:
as mentioned above, you may consider doing this instead (unless I am not misunderstanding something):
```suggestion
    return this.args.totalItems !== undefined;
```
and get rid of the `@hasTotalItems` parameter

Reply:
See my other comments above. "hasTotalItems" is just a UI option to show or hide the total # of items in the UI. (We should consider clearer naming though...)